src/components/Selector.tsx

```diff
@@ -19,7 +21,7 @@ const Wrapper = styled.div`
   }
 `;
 
-type Option = { title: string; id: string; group: string; index: number };
+type Option = { title: string; id: string; group: string; index: number; live: boolean|undefined };
```

Comment:
This can be cleaned up by doing:
```
live?: boolean
```
`?` implies that it's optional (can be undefined)